### PR TITLE
Userdashboard API updated

### PR DESCRIPTION
**Pull request by DhirajGurnani**

### CODE REVIEW

File: app/routes/userRoute/UserRoutes.js [hunk 1]
```diff
@@ -19,6 +19,11 @@ router.post('/login', Controller.login.post);
 
 router.post('/verify-email', Controller.verifyEmail.post);
 
+router.get('/:id/profile', Controller.profile.get);
+router.put('/:id/', Controller.id.put);
+/router.put('/:id/update-password', Controller.updatePassword.put);
```
Comment (rishirajrandive) [inline app/routes/userRoute/UserRoutes.js]: Why is there a / in front of line 24?

File: config/db.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
 'use strict';
 require('dotenv').config();
 let Sequelize = require('sequelize');
+console.log(JSON.stringify(process.env));
```
Comment (rahilvora) [inline config/db.js]: remove the console.

File: app/controllers/userControllers/UserController.js [hunk 3]
```diff
@@ -107,8 +107,100 @@ module.exports = {
...
+                    res.send(response)
+                } else {
+                    let response = {
+                        status_code: 200,
+                        msg: 'data_not_found'
+                    };
+                    res.send(response)
+                }
+
+            });
```
Comment (rahilvora) [inline app/controllers/userControllers/UserController.js]: Add catch statement to handle database error.

File: app/controllers/userControllers/UserController.js [hunk 4]
```diff
@@ -107,8 +107,100 @@ module.exports = {
...
+                    }
+                    res.send(response)
+                } else {
+                    let response = {
+                        status_code: 200,
+                        msg: "records_not_updated_sucessfully"
+                    }
+                    res.send(response)
+                }
+            });
```
Comment (rahilvora) [inline app/controllers/userControllers/UserController.js]: Add catch statement to handle database error.

File: app/controllers/userControllers/UserController.js [hunk 5]
```diff
@@ -107,8 +107,100 @@ module.exports = {
...
+                    };
+                    res.send(response)
+                } else {
+                    let response = {
+                        status_code: 200,
+                        msg: 'data_not_found',
+                    };
+                    res.send(response)
+                }
+            });
```
Comment (rahilvora) [inline app/controllers/userControllers/UserController.js]: Add catch statement to handle database error.

File: app/controllers/userControllers/UserController.js [hunk 6]
```diff
@@ -107,8 +107,100 @@ module.exports = {
...
+                    }
+                    res.send(response)
+                } else {
+                    let response = {
+                        status_code: 200,
+                        msg: "password_not_updated_sucessfully"
+                    }
+                    res.send(response)
+                }
+            });
```
Comment (rahilvora) [inline app/controllers/userControllers/UserController.js]: Add catch statement to handle database error.

**Pull request merged**